packages/paste-core/components/spinner/src/index.tsx

```diff
@@ -22,18 +29,21 @@ const SpinningWrapper = styled.div(
   })
 );
 
-interface SpinnerProps extends SpinnerIconProps {
+interface SpinnerProps extends LoadingIconProps {
   title: string;
+  size?: IconSize;
 }
 
-const Spinner: React.FC<SpinnerProps> = props => (
-  <SpinningWrapper>
-    <SpinnerIcon {...props} />
+const Spinner: React.FC<SpinnerProps> = ({size, ...props}) => (
+  <SpinningWrapper size={size}>
+    <LoadingIcon size={size} {...props} />
```

Comment:
Should we spread props here? Is it only title and size as valid props?

*EDIT:* better question, do we want a whole bunch of different spinners in our apps?

Reply:
I made it explicit.  Good call. https://github.com/twilio-labs/paste/pull/129/commits/ebb040a26e2078448fb6f394a8f915ee4e3bbae4